cl/alerts/tests/tests.py

```diff
@@ -661,6 +661,74 @@ async def test_invalid_alert_type_fail(self) -> None:
         search_alert = Alert.objects.all()
         self.assertEqual(await search_alert.acount(), 0)
 
+    async def test_can_validate_required_fields(self) -> None:
+        # Test creation with missing required fields:
+        response = await self.client.post(self.alert_path, {}, format="json")
+
+        self.assertEqual(response.status_code, HTTPStatus.BAD_REQUEST)
+        self.assertIn("This field is required.", response.json()["name"])
+        self.assertIn("This field is required.", response.json()["query"])
+        self.assertIn("This field is required.", response.json()["rate"])
+
+        # Create a valid alert for subsequent update testing:
+        alert_1 = await self.make_an_alert(
+            self.client,
+            alert_name="alert_1",
+            alert_query=f"q=testing_query&type={SEARCH_TYPES.OPINION}",
+        )
+        alert_1_data = alert_1.json()
+        # Try to update an alert using a PUT request with missing required
+        # fields:
+        alert_1_path_detail = reverse(
+            "alert-detail",
+            kwargs={"pk": alert_1_data["id"], "version": "v3"},
```

Comment:
Hmmm, should this be v4 instead?
```suggestion
            kwargs={"pk": alert_1_data["id"], "version": "v4"},
```

Reply:
You're absolutely right. I'll tweak this right away.